sqldelight-idea-plugin/src/main/kotlin/com/squareup/sqldelight/intellij/inspections/UnusedColumnInspection.kt

```diff
@@ -63,7 +63,10 @@ class UnusedColumnInspection : LocalInspectionTool() {
         .flatMap { file -> file.sqlStmtList?.stmtList.orEmpty() }
         .flatMap { stmt -> stmt.compoundSelectStmt?.queryExposed().orEmpty() }
         .flatMap { queryResult -> queryResult.columns }
-        .forEach { column -> column.element.columnDef()?.let(candidates::remove) }
+        .forEach { column ->
+          val element = column.element.parent
+          element.columnDef()?.let(candidates::remove)
```

Comment:
i think its a bit safer to instead add `if (this is SqlColumnName) return parent.columnDef()` to `PsiElement.columnDef()` since that also catches cases where `element` is already a `SqlColumnDef` - I don't know if that happens in practice but good to be safe

Reply:
makes sense